### PR TITLE
Adds a simple flask application

### PR DESCRIPTION
**Pull request by idealadarsh**
Created a simple flask application as per a GCI 17 task.

### CODE REVIEW

File: idealadarsh/app.py [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+
+app = Flask(__name__)
+
+@app.route("/")
+def index():
+    return render_template('index.html')
+
+@app.route('/result', methods = ['POST'])
+def result():
+    name = request.form['name']
```
Comment (vaibhavsingh97) [inline idealadarsh/app.py:11]: If I press submit button, without entering any name than it's should prompt the user that his/her name is missing

**Review by vaibhavsingh97 — COMMENTED** [review]
@idealadarsh You had done a great job 🎉 🎉 
Just minor feedback so that you can improve. 
Thanks

File: idealadarsh/app.py (hunk 1 above)
Comment (idealadarsh) [reply idealadarsh/app.py:11]: @vaibhavsingh97 Thanks for pointing that out, I'll fix it.

Comment (vaibhavsingh97) [reply idealadarsh/app.py:11]: @idealadarsh Send PR fixing this, so I think before sending another PR you should read https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia/

**Pull request merged**